scalableminds/webknossos
# Search VX workflows

### Steps to test:
- Make sure you have a few vx workers and that vx is connected to webknossos
- go to administration > voxelytics and search for workflow name or hash

### Issues:
- fixes #7730 

------
(Please delete unneeded items, merge only when none are left open)
